python/examples/integrations/flask_streaming/flask_with_whylogs.ipynb

```diff
@@ -0,0 +1,836 @@
+      "{'sepal_length_cm': 6.7, 'sepal_width_cm': 3.3, 'petal_length_cm': 5.7, 'petal_width_cm': 2.5}\n",
+      "{'data': {'class': 'virginica'}, 'message': 'Success'}\n",
+      "{'sepal_length_cm': 6.7, 'sepal_width_cm': 3.0, 'petal_length_cm': 5.2, 'petal_width_cm': 2.3}\n",
+      "{'data': {'class': 'virginica'}, 'message': 'Success'}\n",
+      "{'sepal_length_cm': 6.3, 'sepal_width_cm': 2.5, 'petal_length_cm': 5.0, 'petal_width_cm': 1.9}\n",
+      "{'data': {'class': 'virginica'}, 'message': 'Success'}\n",
+      "{'sepal_length_cm': 6.5, 'sepal_width_cm': 3.0, 'petal_length_cm': 5.2, 'petal_width_cm': 2.0}\n",
+      "{'data': {'class': 'virginica'}, 'message': 'Success'}\n",
+      "{'sepal_length_cm': 6.2, 'sepal_width_cm': 3.4, 'petal_length_cm': 5.4, 'petal_width_cm': 2.3}\n",
+      "{'data': {'class': 'virginica'}, 'message': 'Success'}\n",
+      "{'sepal_length_cm': 5.9, 'sepal_width_cm': 3.0, 'petal_length_cm': 5.1, 'petal_width_cm': 1.8}\n",
+      "{'data': {'class': 'virginica'}, 'message': 'Success'}\n"
+     ]
+    }
+   ],
+   "source": [
+    "healthy = requests.get(f\"{url}/health\")\n",
+    "if healthy.ok:\n",
+    "    for k in range(data_mod.shape[0]):\n",
+    "        # Build a payload with random values\n",
+    "        payload = dict(zip(labels, data_mod.iloc[:, 0:4].values[k]))\n",
+    "        print(payload)\n",
+    "        response = requests.post(f\"{url}/predict\", json=payload)\n",
+    "        if response.ok:\n",
+    "            print(response.json())\n",
+    "            time.sleep(random.randrange(2, 10))"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Viewing the Data\n",
+    "\n",
+    "This app defaults to a local file writer. You can look at the logs within the docker container by running `docker exec -it <container-id> bash` . They will all be in `ls /logs`\n",
+    "\n",
+    "While there you can visualize the results by following the instructions of the [\"Notebook Profile Visualizer\"](https://github.com/whylabs/whylogs/blob/9618e5dd6570bc484579ec1325f2f512ff56977f/python/examples/basic/Notebook_Profile_Visualizer.ipynb)\n",
+    ". Some more information can be found in these notebooks [\"Merging Profiles\"](https://github.com/whylabs/whylogs/blob/9618e5dd6570bc484579ec1325f2f512ff56977f/python/examples/basic/Merging_Profiles.ipynb) and [\"Streaming_Data_with_Log_Rotation\"](https://github.com/whylabs/whylogs/blob/mainline/python/examples/advanced/Log_Rotation_for_Streaming_Data/Streaming_Data_with_Log_Rotation.ipynb).\n",
+    "\n",
+    "## Using WhyLabs\n",
+    "There is another way to look at the visualization by integrating with WhyLabs. Follow the directions in \"WhyLabs Writer\" to change the .env file in this directory. Then rerun the above. This time it will populate your WhyLabs dataset to be able to see the drifts on the WhyLabs portal. \n",
```

Comment:
The first sentence makes it look like WhyLabs is just another way to visualize the profile. Maybe you could change to make it clear that the profiles can be sent to your observability dashboard at WhyLabs directly, or something like that.

Reply:
That's a great point. :)